bin/copyRawDataToPrm.sh

```diff
@@ -605,7 +608,7 @@ else
 			if [[ -e "${controlFileBase}/${rawDataItem}.rsyncRuns.finished" ]]
 			then
 				log4Bash 'TRACE' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "${controlFileBase}/${rawDataItem}.rsyncRuns.finished present."
-				processedRawDataItems=$((${processedRawDataItems}+1))
+				processedRawDataItems=$((processedRawDataItems+1))
```

Comment:
In theorie heeft ShellCheck gelijk dat je in dit soort Bash constructen met`$((...))` geen `${...}` nodig hebt voor variabelen, maar het mag wel en in het kader van consequent gebruik van variabelen zie ik ze liever met. Daarom is SC2004 ook één van de twee, die we globaal disabled hebben in `NGS_Automated/check/shellcheck.sh`